opentelemetry-instrumentation/src/opentelemetry/instrumentation/bootstrap.py

```diff
@@ -46,15 +55,19 @@
     "pyramid": "opentelemetry-instrumentation-pyramid>=0.11b0",
     "redis": "opentelemetry-instrumentation-redis>=0.8b0",
     "requests": "opentelemetry-instrumentation-requests>=0.8b0",
+    "sklearn": "opentelemetry-instrumentation-sklearn>=0.15b0",
     "sqlalchemy": "opentelemetry-instrumentation-sqlalchemy>=0.8b0",
     "sqlite3": "opentelemetry-instrumentation-sqlite3>=0.11b0",
     "starlette": "opentelemetry-instrumentation-starlette>=0.11b0",
+    "system-metrics": "opentelemetry-instrumentation-system-metrics>=0.15b0",
```

Comment:
Should `system-metrics` be part of this?

Reply:
Not sure if it should be! I just went off the folder of [instrumented libraries](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/master/instrumentation) and added it there. Let me know if I should remove it?

Reply:
It should probably be removed since system-metrics isnt instrumenting a particular library

Reply:
Removed!

Reply:
I added a comment saying "system-metrics is not be included" for future reference.